brave/src/main/java/com/linecorp/armeria/common/brave/RequestContextCurrentTraceContext.java

```diff
@@ -142,8 +147,9 @@ private RequestContextCurrentTraceContext(Builder builder) {
     public TraceContext get() {
         final RequestContext ctx = getRequestContextOrWarnOnce();
```

Comment:
this answers my question. a new client context will prioritize vs the thread local one. This is correct imho